.github/copilot-instructions.md

```diff
@@ -0,0 +1,25 @@
+# Repository information
```

Comment:
Still figuring out what information is most helpful. Might want to add something about management plane vs data plane